examples/datasource-basic/README.md

```diff
@@ -1,16 +1,17 @@
-# Grafana Basic Data Source Plugin example
+# Basic Data Source Plugin for Grafana
 
 [![Build](https://github.com/grafana/grafana-starter-datasource/workflows/CI/badge.svg)](https://github.com/grafana/grafana-starter-datasource/actions?query=workflow%3A%22CI%22)
 
-This template is a starting point for building Grafana data source plugins.
+This repository contains a simple data source plugin example.
```

Comment:
Theres duplication between this summary sentence and the Overview section which feels odd to me, but within the panel plugin update the summary was closer to the first sentence in the overview here and provided more context to the plugins purpose.

Reply:
done